### PR TITLE
[Feature] Support Broker Load/Export Table from volcengine tos

### PR DESCRIPTION
**Pull request by Jay-ju**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
   Support Broker Load/Export Table from volcengine tos

- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [x] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

File: fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java [hunk 1]
```diff
@@ -799,6 +812,54 @@ public BrokerFileSystem getCOSFileSystem(String path, Map<String, String> proper
...
+                return null;
+            }
+            if (fileSystem.getDFSFileSystem() == null) {
+                logger.info("could not find file system for path " + path + " create a new one");
+                // create a new filesystem
+                Configuration conf = new Configuration();
+                conf.set(FS_TOS_ACCESS_KEY, accessKey);
+                conf.set(FS_TOS_SECRET_KEY, secretKey);
+                conf.set(FS_TOS_ENDPOINT, endpoint);
+                conf.set(FS_TOS_REGION, region);
```
Comment (dujl) [inline fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java:848]: region 不一这设置，不设置时是否能测试通过？

File: gensrc/thrift/PlanNodes.thrift [hunk 2]
```diff
@@ -175,6 +175,7 @@ enum TObjectStoreType {
   COS,
   OBS,
   UNIVERSAL_FS
```
Comment (dujl) [inline gensrc/thrift/PlanNodes.thrift:177]: tos放UNIVERSAL_FS前面

File: fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java (hunk 1 above)
Comment (Jay-ju) [reply fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java:848]: 可以的

File: gensrc/thrift/PlanNodes.thrift (hunk 2 above)
Comment (Jay-ju) [reply gensrc/thrift/PlanNodes.thrift]: ok

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 53 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/fs/hdfs/HdfsFsManager.java|0|53|00.00%|[381, 382, 1011, 1012, 1013, 1014, 1015, 1016, 1017, 1018, 1019, 1020, 1022, 1023, 1024, 1026, 1027, 1028, 1033, 1034, 1035, 1036, 1037, 1038, 1039, 1041, 1043, 1045, 1048, 1050, 1051, 1053, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1061, 1062, 1063, 1064, 1065, 1066, 1068, 1069, 1070, 1073, 1074, 1075, 1076, 1078]|

**Comment by xiaoyong-z** [comment]
Thank you! 
I think you should manually THdfsProperties tProperties in HdfsFsManager.java `convertObjectStoreConfToProperties`  to support `broker load (without broker in 2.5)`

**Comment by xiaoyong-z** [comment]
> Thank you! I think you should manually THdfsProperties tProperties in HdfsFsManager.java `convertObjectStoreConfToProperties` to support `broker load (without broker in 2.5)`

image
In 2.5, we don't  need to deply the broker to load data from HDFS or S3

**Comment by Jay-ju** [comment]
> 53

done

**Comment by Jay-ju** [comment]
> #### [FE PR Coverage Check]
> 😞 **fail** : 0 / 53 (00.00%)
> 
> #### file detail
> path	covered_line	new_line	coverage	not_covered_line_detail
> 🔵	com/starrocks/fs/hdfs/HdfsFsManager.java	0	53	00.00%	[381, 382, 1011, 1012, 1013, 1014, 1015, 1016, 1017, 1018, 1019, 1020, 1022, 1023, 1024, 1026, 1027, 1028, 1033, 1034, 1035, 1036, 1037, 1038, 1039, 1041, 1043, 1045, 1048, 1050, 1051, 1053, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1061, 1062, 1063, 1064, 1065, 1066, 1068, 1069, 1070, 1073, 1074, 1075, 1076, 1078]

this test depends on  Regression Testing， need some tos env, so i don't add ut here

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 68 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/fs/hdfs/HdfsFsManager.java|0|68|00.00%|[89, 90, 91, 92, 211, 213, 214, 216, 217, 219, 220, 221, 223, 224, 226, 404, 405, 1034, 1035, 1036, 1037, 1038, 1039, 1040, 1041, 1042, 1043, 1045, 1046, 1047, 1049, 1050, 1051, 1056, 1057, 1058, 1059, 1060, 1061, 1062, 1064, 1066, 1068, 1071, 1073, 1074, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1086, 1087, 1088, 1089, 1091, 1092, 1093, 1096, 1097, 1098, 1099, 1101]|

**Review by xiaoyong-z — APPROVED** [review]

**Comment by xiaoyong-z** [comment]
run starrocks_fe_unittest

**Comment by xiaoyong-z** [comment]
run all

**Comment by xiaoyong-z** [comment]
run all

**Comment by xiaoyong-z** [comment]
cc @chaoyli @meegoo @gengjun-git

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 68 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/fs/hdfs/HdfsFsManager.java|0|68|00.00%|[89, 90, 91, 92, 211, 213, 214, 216, 217, 219, 220, 221, 223, 224, 226, 404, 405, 1034, 1035, 1036, 1037, 1038, 1039, 1040, 1041, 1042, 1043, 1045, 1046, 1047, 1049, 1050, 1051, 1056, 1057, 1058, 1059, 1060, 1061, 1062, 1064, 1066, 1068, 1071, 1073, 1074, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1086, 1087, 1088, 1089, 1091, 1092, 1093, 1096, 1097, 1098, 1099, 1101]|

**Review by meegoo — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by xiaoyong-z** [comment]
run starrocks_admit_test

**Pull request merged**